scripts/Interface/ui/reflectometer/data_refl_simple.ui

```diff
@@ -0,0 +1,4648 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As above, this file can be removed. There's also a file `buildconfig/ui_data_refl_simple.ui` which looks like a stray file that should be removed.